internal/fakecgo/trampolines_arm64.s

```diff
@@ -10,29 +10,24 @@
 // these trampolines map the gcc ABI to Go ABI and then calls into the Go equivalent functions.
```

Comment:
Like this comment says this is technically converting from C to Go ABI. Since in this case the arguments get placed in the same spot is it a good idea to leave a comment say R0, R1 are already in the right spot for Go's ABI?